src/components/formComponentTemplateTxnOps.tsx

```diff
@@ -110,6 +95,22 @@ type TemplateRenderClaimantsProps = {
   isRequired?: boolean;
 };
 
+type TemplateRenderSorobanProps = {
+  value: string | undefined;
+  error: string | undefined;
+  onChange: (val: any) => void;
+  isRequired?: boolean;
+  isDisabled?: boolean;
+};
+
+type TemplateRenderSorobanInvokeProps = {
+  value: SorobanInvokeValue | undefined;
+  error: string | undefined;
+  onChange: (val: SorobanInvokeValue | undefined) => void;
```

Comment:
this file just includes name changes and updating param value for `onChange` from just string to an object